packages/zend-locale/library/Zend/Locale/Math.php

```diff
@@ -216,11 +216,12 @@ public static function localize($value)
      * Fixes a problem of BCMath with numbers containing exponents
      *
      * @param integer $value Value to erase the exponent
-     * @param integer $scale (Optional) Scale to use
+     * @param int|null $scale (Optional) Scale to use
      * @return string
      */
     public static function exponent($value, $scale = null)
     {
+        $scale = $scale > 0 ? $scale : 0;
```

Comment:
```suggestion
        $scale = max(0, $scale);
```

what you think of using max? and/or perhaps use (int) casting?

Reply:
Casting would change the behaviour as it would remove any warnings and not return null when using anything else than an int(ish) value